scripts/convert_svd_to_diffusers.py

```diff
@@ -728,3 +733,42 @@ def convert_ldm_vae_checkpoint(checkpoint, config):
     assign_to_checkpoint(paths, new_checkpoint, vae_state_dict, additional_replacements=[meta_path], config=config)
     conv_attn_to_linear(new_checkpoint)
     return new_checkpoint
+
+
+def read_config_file(filename):
+    # The yaml file contains annotations that certain values should
+    # loaded as tuples.
+    with open(filename) as f:
+        original_config = yaml.load(f, FullLoader)
+
+    return original_config
+
+
+if __name__ == "__main__":
+    
+
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--checkpoint_path", default="stabilityai/stable-video-diffusion-img2vid-xt-1-1/svd_xt_1_1.safetensors", type=str, help="Path to the checkpoint to convert."
+    )
+    parser.add_argument(
+        "--config_file",
+        default="generative-models/scripts/sampling/configs/svd_xt_image_decoder.yaml",
+        type=str,
+        help="The config json file corresponding to the architecture.",
+    )
+    parser.add_argument("--dump_path", default=None, type=str, help="Path to the output model.")
+    args = parser.parse_args()
+    
+    args.config_file = read_config_file(args.config_file)
+    args.config_file = create_unet_diffusers_config(args.config_file, 768)
+
+    state_dict = load_file(args.checkpoint_path)
+    unet = convert_ldm_unet_checkpoint(state_dict,args.config_file)
+    from diffusers import  UNetSpatioTemporalConditionModel
+    unet_svd = UNetSpatioTemporalConditionModel.from_pretrained("stabilityai/stable-video-diffusion-img2vid-xt")
+    diff_0 = set(unet_svd.state_dict().keys()) - set(unet.keys())
+    print(diff_0)
+    unet_svd.load_state_dict(unet)
+    unet_svd.save_pretrained("stabilityai/stable-video-diffusion-img2vid-xt-1-1/svd_xt_1_1")
```

Comment:
Here, we should save the whole pipeline.
The path to save should be accepted as an argument.